src/auth/oauth/test/oauth.test.ts

```diff
@@ -370,21 +370,11 @@ describe('validateAuthCallback', () => {
     testCallbackQuery.hmac = expectedHmac;
 
     fetchMock.mockResponse(JSON.stringify(successResponse));
-    await ShopifyOAuth.validateAuthCallback(req, res, testCallbackQuery);
+    const returnedCookie = await ShopifyOAuth.validateAuthCallback(req, res, testCallbackQuery);
 
-    let expectedCookieExpiration = Date.now() / 1000;
-    expectedCookieExpiration += 30;
     const cookieSession = await Context.SESSION_STORAGE.loadSession(cookies.id);
     expect(cookieSession).not.toBeUndefined();
-
-    if (cookieSession?.expires) {
-      const actualCookieExpiration: number =
-        cookieSession.expires.getTime() / 1000;
-      // 1-second grace period
-      expect(
-        Math.abs(expectedCookieExpiration - actualCookieExpiration),
-      ).toBeLessThan(1);
-    }
+    expect(cookieSession).toEqual(returnedCookie);
```

Comment:
This session should actually be deleted now - `loadSession` will return it but the expiration should be in the past.